BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -94,6 +94,7 @@ public final class ConvivaAnalytics: NSObject {
 
         listener = BitmovinPlayerListener(player: player)
         listener?.delegate = self
+        adAnalytics.setUpdateHandler(handleAdUpdateRequest)
```

Comment:
Do you know when this handler is called? 🤔

Reply:
I don't know the exact schedule, but it is the recommended way of reporting those metrics.